swf/src/types/blur_filter.rs

```diff
@@ -31,10 +31,10 @@ impl BlurFilter {
         self.num_passes() == 0 || (self.blur_x <= Fixed16::ONE && self.blur_y <= Fixed16::ONE)
     }
 
-    pub fn calculate_dest_rect(&self, source_rect: Rectangle<i32>) -> Rectangle<i32> {
+    pub fn calculate_dest_rect(&self, source_rect: Rectangle<Twips>) -> Rectangle<Twips> {
         let scale = PASS_SCALES[self.num_passes().clamp(1, 15) as usize - 1];
-        let x = (scale * self.blur_x.to_f32()).ceil().max(0.0) as i32;
-        let y = (scale * self.blur_y.to_f32()).ceil().max(0.0) as i32;
+        let x = Twips::from_pixels(scale * self.blur_x.to_f64());
+        let y = Twips::from_pixels(scale * self.blur_y.to_f64());
```

Comment:
Is the `.ceil().max(0.0)` here not necessary anymore? Or was this an accidental change?

Reply:
Hmm, max should stay, but ceil is handled by twips->int

Reply:
Fixed!